File: inventory/scripts/python/simulation_mode.py
def handle_simulation_mode(inFiles, outFiles):
    """
    Handle simulation mode for data processing scripts.
    Uses standard wrappers with simulation mode.
    """
    import sys
    import os

    global odeReadParquet, odeWriteParquet
    
    if (os.getenv("AIR_SIMULATION_MODE") == "true"):
        # Exit the script
        sys.exit(0)

File: inventory/scripts/R/simulation_mode.R
handle_simulation_mode <- function(inFiles, outFiles) {
    # Check if simulation mode is enabled
    if (tolower(Sys.getenv("AIR_SIMULATION_MODE", "false")) == "true") {
        # Immediately exit the script
        quit(save = "no", status = 0)
    }
}